Makefile

```diff
@@ -222,12 +223,21 @@ undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/confi
 	$(KUSTOMIZE) build config/default | kubectl delete -f -
 
 install-authorino: install-cert-manager ## install RBAC and CRD for authorino
+	kubectl create namespace authorino-operator
 	$(KUSTOMIZE) build config/authorino | kubectl apply -f -
 
+uninstall-authorino:  ## uninstall RBAC and CRD for authorino
+	$(KUSTOMIZE) build config/authorino | kubectl delete -f -
+	kubectl delete namespace authorino-operator
```

Comment:
Deleting the operator's namespace seems a bit harsh. Effectively, this will decommission the operator instance. IWO, it's almost like a "undeploy" target.

Reply:
Ya the namespace thing was something that kept biting me when doing this. As the create of the namespace is documented in the README I would be happy the remove the changes I made around the namespace.